CHANGELOG.md

```diff
@@ -1,439 +1,449 @@
-- BREAKING: Rename `stub_account_api_delete_saved_page_unauthorised` to `stub_account_api_unauthorized_delete_saved_page` for consistency (for Account API)
-- BREAKING: Change how we apply Sentry's fingerprinting algorithm ([#1096](https://github.com/alphagov/gds-api-adapters/pull/1096)).
+* BREAKING: Remove `create_registration_state` method and helpers (for Account API)
+* BREAKING: Remove `state_id` parameter from auth URL method and helpers (for Account API)
+* BREAKING: Remove `check_for_email_subscription` method and helpers (for Account API)
+* BREAKING: Remove `set_email_subscription` method and helpers (for Account API)
+* BREAKING: Rename `stub_account_api_get_email_subscription_unauthorized` to `stub_account_api_unauthorized_get_email_subscription` for consistency (for Account API)
+* BREAKING: Rename `stub_account_api_delete_saved_page_unauthorised` to `stub_account_api_unauthorized_delete_saved_page` for consistency (for Account API)
+* BREAKING: Change how we apply Sentry's fingerprinting algorithm ([##1096](https://github.com/alphagov/gds-api-adapters/pull/1096)).
   Updating gds-api-adapters but failing to update govuk_app_config to v4 may mean application errors are not as effectively 'grouped' in Sentry.
 
-# 71.9.0
+## 71.9.0
 
-- Add `cookie_consent` to Account API auth callback test helper
+* Add `cookie_consent` to Account API auth callback test helper
 
-# 71.8.0
+## 71.8.0
 
-- Add `on_conflict` parameter to `change_subscriber` (for Email Alert API)
-- Add test helper for Email Alert API auth tokens for GOV.UK Account users
+* Add `on_conflict` parameter to `change_subscriber` (for Email Alert API)
+* Add test helper for Email Alert API auth tokens for GOV.UK Account users
 
-# 71.7.0
+## 71.7.0
 
 * Add `id` to Account API user info test helper.
 * Add `govuk_account_id` to Email Alert API subscriber test helpers.
 * Add `link_subscriber_to_govuk_account` (for Email Alert API)
 * Add `find_subscriber_by_govuk_account` (for Email Alert API)
 
-# 71.6.0
+## 71.6.0
 
 * Add `authenticate_subscriber_by_govuk_account` (for Email Alert API)
 
-# 71.5.0
+## 71.5.0
 
 * Add functions and test helpers for account-linked email subscriptions.
 * Add `has_unconfirmed_email` option to `update_user_by_subject_identifier`.
 * Add `has_unconfirmed_email` option to `get_user` test helper.
 
-# 71.4.0
+## 71.4.0
 
 * Add `update_user_by_subject_identifier` (for Account API)
 * Update "save a page" helpers to use new routes.
 
-# 71.3.0
+## 71.3.0
 
 * Add `content_id` and `title` fields to "save a page" test helpers.
 * Add `get_user` (for Account API)
 
-# 71.2.0
+## 71.2.0
 
 * Add endpoints for authenticated "Save a page" API
 
-# 71.1.0
+## 71.1.0
 
-* Removed `LinkCheckerApi#upsert_resource_monitor` method as it’s not supported by Link Checker API itself.
+* Removed `LinkCheckerApi##upsert_resource_monitor` method as it’s not supported by Link Checker API itself.
```

Comment:
Was this change intentional?  Similar question for release 61.0.0 and a few others.

Reply:
Oops